client/src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import 'antd/dist/antd.css';
```

Comment:
please import tine antd.css inside the css file and pay attention to import order
you need to overwrite the antd.css so put at the top then make your own css